cli/command/image/build_test.go

```diff
@@ -68,3 +68,24 @@ func TestRunBuildDockerfileFromStdinWithCompress(t *testing.T) {
 	sort.Strings(actual)
 	assert.Equal(t, []string{dockerfileName, ".dockerignore", "foo"}, actual)
 }
+
+// TestRunBuildFromLocalGitHubDirNonExistingRepo tests that build contexts
+// starting with `github.com/` are special-cased, and the build command attempts
+// to clone the remote repo, however a local directory should take precedence.
+func TestRunBuildFromLocalGitHubDirNonExistingRepo(t *testing.T) {
```

Comment:
Was doubting; in a single test, it more "proves" that without local directory it does X, and with it does Y, but guess it doesn't matter if split; will update